templates/vue-demo-store/components/listing-filters/ListingFiltersProperties.vue

```diff
@@ -1,21 +1,16 @@
-<script
-  setup
-  lang="ts"
-  generic="
-    ListingFilter extends {
-      code: string;
-      label: string;
-      name: string;
-      options: Array<Schemas['PropertyGroupOption']>;
-    }
-  "
->
+<script setup lang="ts">
 import { inject, ref } from "vue";
 import { getTranslatedProperty } from "@shopware-pwa/helpers-next";
 import type { Schemas } from "#shopware";
 
 defineProps<{
-  filter: ListingFilter;
+  filter: {
```

Comment:
Why have you switched from generic? It might cause type problems if anyone would like to use this component directly.